docs/misc/operation/chaindata-migration.md

```diff
@@ -1,30 +1,3 @@
-# Migrate Chaindata
+# State Migration how to
```

Comment:
Can we reword this to: State Migration: A Step-by-Step Guide ?

Reply:
This page is merely a redirection. Let me revert to the original title.